hadoop/src/main/java/org/apache/carbondata/hadoop/api/CarbonOutputCommitter.java

```diff
@@ -168,6 +168,9 @@ public void commitJob(JobContext context) throws IOException {
           throw new IOException(e);
         }
       }
+      // After merging index, update newMetaEntry with updated data and index size
```

Comment:
Line 156 also calling the API , now that values will be overwritten by this. so previous call is not required. Find segment is empty or not by some other API

Reply:
that call is before merging the index file right, so once job is committed, we need to update the the size, and then after merge index we need to again do it.

Reply:
Added new API to get index size alone for the segment and updated index size in LoadMetaData entry after merge index event